src/OpenCvSharp/Modules/core/Struct/Size.cs

```diff
@@ -73,22 +73,32 @@ public Size(double width, double height)
         return !lhs.Equals(rhs);
     }
 
+    public static implicit operator Size(Size2d size)
+    {
+        return new(size.Width, size.Height);
```

Comment:
I think `Size -> Size2d` is OK, but `Size2d -> Size` should be marked as `explicit`. This is because there is no implicit conversion from double to int in the case of scalar values. (For example, `double d = 1;` is OK but `int i = 1.23;` causes an error.)

https://github.com/shimat/opencvsharp/blob/0498156821f052ff6b2fd1205148f07a1de2781d/src/OpenCvSharp/Modules/core/Struct/Point2d.cs#L43